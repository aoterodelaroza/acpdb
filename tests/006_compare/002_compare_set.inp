## check: 002_compare_set.out -a1e-10
## delete: 002_compare_set.db
## labels: regression quick

system rm -f 002_compare_set.db
connect 002_compare_set.db

insert method test
end
insert method test2
end

insert litref bibtex ../zz_source/bsip2.bib
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end
insert set s22b
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end

insert calc
 property_type energy_difference
 file ../zz_source/s22.list
 method test2
end
insert calc
 property_type energy_difference
 file ../zz_source/s22b.list
 method test2
end


delete evaluation test2 s22.nh3_nh3

compare
 property_type energy_difference
 method test2
 file ../zz_source/s22.list2
end
compare
 property_type energy_difference
 method test2
 file ../zz_source/s22.list2
 set s22
end

compare
 property_type energy_difference
 method test2
 file ../zz_source/s22all.list2
end
