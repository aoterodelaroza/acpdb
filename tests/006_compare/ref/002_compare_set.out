%% verbose
%% system rm -f 002_compare_set.db
* SYSTEM: rm -f 002_compare_set.db

%% connect 002_compare_set.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 002_compare_set.db
Creating skeleton database 

%% insert method test
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

%% insert method test2
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test2

%% insert litref bibtex ../zz_source/bsip2.bib
* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

%% insert set s22
* INSERT: insert data into the database (SET)
# INSERT SET s22

%% insert set s22b
* INSERT: insert data into the database (SET)
# INSERT SET s22b

%% insert calc
* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test2;property=22;nvalue=1)
# INSERT EVALUATION (method=test2;property=21;nvalue=1)
# INSERT EVALUATION (method=test2;property=20;nvalue=1)
# INSERT EVALUATION (method=test2;property=19;nvalue=1)
# INSERT EVALUATION (method=test2;property=18;nvalue=1)
# INSERT EVALUATION (method=test2;property=17;nvalue=1)
# INSERT EVALUATION (method=test2;property=16;nvalue=1)
# INSERT EVALUATION (method=test2;property=15;nvalue=1)
# INSERT EVALUATION (method=test2;property=14;nvalue=1)
# INSERT EVALUATION (method=test2;property=1;nvalue=1)
# INSERT EVALUATION (method=test2;property=2;nvalue=1)
# INSERT EVALUATION (method=test2;property=3;nvalue=1)
# INSERT EVALUATION (method=test2;property=4;nvalue=1)
# INSERT EVALUATION (method=test2;property=5;nvalue=1)
# INSERT EVALUATION (method=test2;property=6;nvalue=1)
# INSERT EVALUATION (method=test2;property=7;nvalue=1)
# INSERT EVALUATION (method=test2;property=8;nvalue=1)
# INSERT EVALUATION (method=test2;property=9;nvalue=1)
# INSERT EVALUATION (method=test2;property=10;nvalue=1)
# INSERT EVALUATION (method=test2;property=11;nvalue=1)
# INSERT EVALUATION (method=test2;property=12;nvalue=1)
# INSERT EVALUATION (method=test2;property=13;nvalue=1)

%% insert calc
* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test2;property=44;nvalue=1)
# INSERT EVALUATION (method=test2;property=43;nvalue=1)
# INSERT EVALUATION (method=test2;property=42;nvalue=1)
# INSERT EVALUATION (method=test2;property=41;nvalue=1)
# INSERT EVALUATION (method=test2;property=40;nvalue=1)
# INSERT EVALUATION (method=test2;property=39;nvalue=1)
# INSERT EVALUATION (method=test2;property=38;nvalue=1)
# INSERT EVALUATION (method=test2;property=37;nvalue=1)
# INSERT EVALUATION (method=test2;property=36;nvalue=1)
# INSERT EVALUATION (method=test2;property=23;nvalue=1)
# INSERT EVALUATION (method=test2;property=24;nvalue=1)
# INSERT EVALUATION (method=test2;property=25;nvalue=1)
# INSERT EVALUATION (method=test2;property=26;nvalue=1)
# INSERT EVALUATION (method=test2;property=27;nvalue=1)
# INSERT EVALUATION (method=test2;property=28;nvalue=1)
# INSERT EVALUATION (method=test2;property=29;nvalue=1)
# INSERT EVALUATION (method=test2;property=30;nvalue=1)
# INSERT EVALUATION (method=test2;property=31;nvalue=1)
# INSERT EVALUATION (method=test2;property=32;nvalue=1)
# INSERT EVALUATION (method=test2;property=33;nvalue=1)
# INSERT EVALUATION (method=test2;property=34;nvalue=1)
# INSERT EVALUATION (method=test2;property=35;nvalue=1)

%% delete evaluation test2 s22.nh3_nh3
* DELETE: delete data from the database (EVALUATION)

# DELETE EVALUATION (method=test2;property=s22.nh3_nh3)

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from file -- 
# File: ../zz_source/s22.list2
# Property type: ENERGY_DIFFERENCE
# Reference method: test2
# Statistics: (partial, missing: 24 from source, 1 from reference)
# s22              rms = 157.04241475   mae = 142.01530111   mse = -142.01530111  wrms = 684.53201574  ndat = 19
# ALL              rms = 157.04241475   mae = 142.01530111   mse = -142.01530111  wrms = 684.53201574  ndat = 19
#
Id Name                                                File         Ref_method         difference
1  s22.h2o_h2o                               -4518.0681840000   -4392.5662900000    -125.5018940000
2  s22.h2co2_h2co2                           -6275.0947000000   -6275.0947000000       0.0000000000
3  s22.formamide_formamide                   -8283.1250040000   -8157.6231100000    -125.5018940000
4  s22.uracil_uracil_hb                     -10228.4043610000  -10040.1515200000    -188.2528410000
5  s22.pyridoxine_aminopyridine             -12048.1818240000  -11922.6799300000    -125.5018940000
6  s22.adenine_thymine_wcc1                 -14056.2121280000  -13805.2083400000    -251.0037880000
7  s22.ch4_ch4                              -15813.2386440000  -15687.7367500000    -125.5018940000
8  s22.c2h4_c2h4                            -17695.7670540000  -17570.2651600000    -125.5018940000
9  s22.c6h6_ch4                             -19452.7935700000  -19452.7935700000       0.0000000000
10 s22.c6h6_c6h6_pd                         -21460.8238740000  -21335.3219800000    -125.5018940000
11 s22.pyrazine_pyrazine                    -23406.1032310000  -23217.8503900000    -188.2528410000
12 s22.uracil_uracil_stack                  -25225.8806940000  -25100.3788000000    -125.5018940000
13 s22.indole_c6h6_stack                    -27233.9109980000  -26982.9072100000    -251.0037880000
14 s22.c2h4_c2h2                            -30873.4659240000  -30747.9640300000    -125.5018940000
15 s22.c6h6_nh3                             -34638.5227440000  -34513.0208500000    -125.5018940000
16 s22.c6h6_hcn                             -36583.8021010000  -36395.5492600000    -188.2528410000
17 s22.c6h6_c6h6_t                          -38403.5795640000  -38278.0776700000    -125.5018940000
18 s22.indole_c6h6_t                        -40411.6098680000  -40160.6060800000    -251.0037880000
19 s22.phenol_phenol                        -42168.6363840000  -42043.1344900000    -125.5018940000
## The following properties are missing from the REFERENCE:
## s22.nh3_nh3
## The following properties are missing from the FILE:
## s22.adenine_thymine_stack
## s22.c6h6_h2o
## s22b.nh3_nh3
## s22b.h2o_h2o
## s22b.h2co2_h2co2
## s22b.formamide_formamide
## s22b.uracil_uracil_hb
## s22b.pyridoxine_aminopyridine
## s22b.adenine_thymine_wcc1
## s22b.ch4_ch4
## s22b.c2h4_c2h4
## s22b.c6h6_ch4
## s22b.c6h6_c6h6_pd
## s22b.pyrazine_pyrazine
## s22b.uracil_uracil_stack
## s22b.indole_c6h6_stack
## s22b.adenine_thymine_stack
## s22b.c2h4_c2h2
## s22b.c6h6_h2o
## s22b.c6h6_nh3
## s22b.c6h6_hcn
## s22b.c6h6_c6h6_t
## s22b.indole_c6h6_t
## s22b.phenol_phenol

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from file -- 
# File: ../zz_source/s22.list2
# Property type: ENERGY_DIFFERENCE
# Reference method: test2
# Statistics: (partial, missing: 2 from source, 1 from reference)
# s22              rms = 157.04241475   mae = 142.01530111   mse = -142.01530111  wrms = 684.53201574  ndat = 19
# ALL              rms = 157.04241475   mae = 142.01530111   mse = -142.01530111  wrms = 684.53201574  ndat = 19
#
Id Name                                                File         Ref_method         difference
1  s22.h2o_h2o                               -4518.0681840000   -4392.5662900000    -125.5018940000
2  s22.h2co2_h2co2                           -6275.0947000000   -6275.0947000000       0.0000000000
3  s22.formamide_formamide                   -8283.1250040000   -8157.6231100000    -125.5018940000
4  s22.uracil_uracil_hb                     -10228.4043610000  -10040.1515200000    -188.2528410000
5  s22.pyridoxine_aminopyridine             -12048.1818240000  -11922.6799300000    -125.5018940000
6  s22.adenine_thymine_wcc1                 -14056.2121280000  -13805.2083400000    -251.0037880000
7  s22.ch4_ch4                              -15813.2386440000  -15687.7367500000    -125.5018940000
8  s22.c2h4_c2h4                            -17695.7670540000  -17570.2651600000    -125.5018940000
9  s22.c6h6_ch4                             -19452.7935700000  -19452.7935700000       0.0000000000
10 s22.c6h6_c6h6_pd                         -21460.8238740000  -21335.3219800000    -125.5018940000
11 s22.pyrazine_pyrazine                    -23406.1032310000  -23217.8503900000    -188.2528410000
12 s22.uracil_uracil_stack                  -25225.8806940000  -25100.3788000000    -125.5018940000
13 s22.indole_c6h6_stack                    -27233.9109980000  -26982.9072100000    -251.0037880000
14 s22.c2h4_c2h2                            -30873.4659240000  -30747.9640300000    -125.5018940000
15 s22.c6h6_nh3                             -34638.5227440000  -34513.0208500000    -125.5018940000
16 s22.c6h6_hcn                             -36583.8021010000  -36395.5492600000    -188.2528410000
17 s22.c6h6_c6h6_t                          -38403.5795640000  -38278.0776700000    -125.5018940000
18 s22.indole_c6h6_t                        -40411.6098680000  -40160.6060800000    -251.0037880000
19 s22.phenol_phenol                        -42168.6363840000  -42043.1344900000    -125.5018940000
## The following properties are missing from the REFERENCE:
## s22.nh3_nh3
## The following properties are missing from the FILE:
## s22.adenine_thymine_stack
## s22.c6h6_h2o

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from file -- 
# File: ../zz_source/s22all.list2
# Property type: ENERGY_DIFFERENCE
# Reference method: test2
# Statistics: (partial, missing: 4 from source, 1 from reference)
# s22              rms = 157.04241475   mae = 142.01530111   mse = -142.01530111  wrms = 684.53201574  ndat = 19
# s22b             rms = 155.61728792   mae = 141.18963075   mse = -141.18963075  wrms = 695.94166852  ndat = 20
# ALL              rms = 156.31320355   mae = 141.59188041   mse = -141.59188041  wrms = 976.17564329  ndat = 39
#
Id Name                                                File         Ref_method         difference
1  s22.h2o_h2o                               -4518.0681840000   -4392.5662900000    -125.5018940000
2  s22.h2co2_h2co2                           -6275.0947000000   -6275.0947000000       0.0000000000
3  s22.formamide_formamide                   -8283.1250040000   -8157.6231100000    -125.5018940000
4  s22.uracil_uracil_hb                     -10228.4043610000  -10040.1515200000    -188.2528410000
5  s22.pyridoxine_aminopyridine             -12048.1818240000  -11922.6799300000    -125.5018940000
6  s22.adenine_thymine_wcc1                 -14056.2121280000  -13805.2083400000    -251.0037880000
7  s22.ch4_ch4                              -15813.2386440000  -15687.7367500000    -125.5018940000
8  s22.c2h4_c2h4                            -17695.7670540000  -17570.2651600000    -125.5018940000
9  s22.c6h6_ch4                             -19452.7935700000  -19452.7935700000       0.0000000000
10 s22.c6h6_c6h6_pd                         -21460.8238740000  -21335.3219800000    -125.5018940000
11 s22.pyrazine_pyrazine                    -23406.1032310000  -23217.8503900000    -188.2528410000
12 s22.uracil_uracil_stack                  -25225.8806940000  -25100.3788000000    -125.5018940000
13 s22.indole_c6h6_stack                    -27233.9109980000  -26982.9072100000    -251.0037880000
14 s22.c2h4_c2h2                            -30873.4659240000  -30747.9640300000    -125.5018940000
15 s22.c6h6_nh3                             -34638.5227440000  -34513.0208500000    -125.5018940000
16 s22.c6h6_hcn                             -36583.8021010000  -36395.5492600000    -188.2528410000
17 s22.c6h6_c6h6_t                          -38403.5795640000  -38278.0776700000    -125.5018940000
18 s22.indole_c6h6_t                        -40411.6098680000  -40160.6060800000    -251.0037880000
19 s22.phenol_phenol                        -42168.6363840000  -42043.1344900000    -125.5018940000
20 s22b.nh3_nh3                              -2635.5397740000   -2510.0378800000    -125.5018940000
21 s22b.h2o_h2o                              -4518.0681840000   -4392.5662900000    -125.5018940000
22 s22b.h2co2_h2co2                          -6275.0947000000   -6275.0947000000       0.0000000000
23 s22b.formamide_formamide                  -8283.1250040000   -8157.6231100000    -125.5018940000
24 s22b.uracil_uracil_hb                    -10228.4043610000  -10040.1515200000    -188.2528410000
25 s22b.pyridoxine_aminopyridine            -12048.1818240000  -11922.6799300000    -125.5018940000
26 s22b.adenine_thymine_wcc1                -14056.2121280000  -13805.2083400000    -251.0037880000
27 s22b.ch4_ch4                             -15813.2386440000  -15687.7367500000    -125.5018940000
28 s22b.c2h4_c2h4                           -17695.7670540000  -17570.2651600000    -125.5018940000
29 s22b.c6h6_ch4                            -19452.7935700000  -19452.7935700000       0.0000000000
30 s22b.c6h6_c6h6_pd                        -21460.8238740000  -21335.3219800000    -125.5018940000
31 s22b.pyrazine_pyrazine                   -23406.1032310000  -23217.8503900000    -188.2528410000
32 s22b.uracil_uracil_stack                 -25225.8806940000  -25100.3788000000    -125.5018940000
33 s22b.indole_c6h6_stack                   -27233.9109980000  -26982.9072100000    -251.0037880000
34 s22b.c2h4_c2h2                           -30873.4659240000  -30747.9640300000    -125.5018940000
35 s22b.c6h6_nh3                            -34638.5227440000  -34513.0208500000    -125.5018940000
36 s22b.c6h6_hcn                            -36583.8021010000  -36395.5492600000    -188.2528410000
37 s22b.c6h6_c6h6_t                         -38403.5795640000  -38278.0776700000    -125.5018940000
38 s22b.indole_c6h6_t                       -40411.6098680000  -40160.6060800000    -251.0037880000
39 s22b.phenol_phenol                       -42168.6363840000  -42043.1344900000    -125.5018940000
## The following properties are missing from the REFERENCE:
## s22.nh3_nh3
## The following properties are missing from the FILE:
## s22.adenine_thymine_stack
## s22.c6h6_h2o
## s22b.adenine_thymine_stack
## s22b.c6h6_h2o

